docs/home/saas-updates.md

```diff
@@ -8,6 +8,19 @@ Latest GA release notes ([https://docs.run.ai/latest/home/whats-new-2-19/](https
 
 SaaS features are gradually rolled out to customers over the course of a week to ensure a smooth transition and minimize any potential disruption. 
 
+## December Release 
+
+### Resolved Bugs 
+
+| ID | Description |
+| :---- | :---- |
+| RUN-23583 | Fixed a bug where navigating through the context menu in the new UI could cause some pages to incorrectly appear as active |
+| RUN-23471 | Resolved a bug where deleting an environment variable from an environment asset prevented it from being re-added |
+| RUN-23446 | Fixed a bug where the node pool in quota management could display incorrectly when creating a new project | 
```

Comment:
| RUN-23446 | Fixed a bug where the node pool in quota management could be displayed incorrectly when creating a new project |